feature-utils/poly-look/src/react-components/cards/clickableCard.jsx

```diff
@@ -24,12 +24,11 @@ const ClickableCard = ({
     >
       {children}
       {buttonText && (
-        <button
-          className="poly-button centered"
+        <PolyButton
+          className="centered"
```

Comment:
FWIW, I'd have preferred it if we were to just use the built-in `button` tag for buttons. Previously we were putting some `poly-button` class on them manually - but why even do that? We can just style `button` tags globally with polyLook's CSS, and give it some behaviour, too.

But I'm OK with this, maybe it's easier this way. But I would consider not having to provide some weird CSS class, but rather have some `centered=true` prop here.

Reply:
The thing with `centered` is that I've seen it used in some other places and it looked to me like an utility class. Maybe I should have made that a bit more explicit and put it under `css/utils.css`. But, a `centered=true` works for me as well.

Reply:
Yeah, we're a bit inconsistent here, but if we actually want this library to be reusable outside of our context, we really have to be more conscious about interface vs implementation details and polluting the global namespace and such things. Do we explicitly expose a `centered` CSS class? Works for me I guess - bit more arcane than props I guess, but it can be used universally. But we should totally prefix that IMHO, it's pretty likely someone else is going to create a CSS class called `centered` and then all hell might break lose...

But since you're not introducing that class here, it might be best if you just leave it alone. Wouldn't insist on a prop either.

Reply:
Done.